templates/todo/projects/csharp-cosmos-sql/README.md

```diff
@@ -41,7 +41,7 @@ azd up --template todo-csharp-cosmos-sql
 
 You will be prompted for the following information:
 
-- `Environment Name`: This will be used as a prefix for all your Azure resources, make sure it is globally unique and under 15 characters.
+- `Environment Name`: This will be used as a prefix for the resource group that will be created to hold all Azure resources.
```

Comment:
@puicchan - I think that we might want to call out that the environment name should be unique within your subscription, if someone tries to deploy two different templates to the same environment name in the same subscription, I think they will clash with one another.  These no longer need to be globally unique however.

Reply:
Adding "This name should be unique within your Azure subscription".